ml_tools/eolearn/ml_tools/train_split.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tasks used for train set preparation.
+"""
+import numpy as np
+
+from eolearn.core import EOTask
+
+
+class TrainSplitTask(EOTask):
+    """ Randomly assigns each value to a class of values by generating a class mask.
```

Comment:
Can the documentation be more explicit? Suggestion:

Randomly assign each or group of pixels to a train, test, and/or validation sets. The group of pixels is defined by an input feature (i.e. MASK_TIMELESS with polygon ids, or connected component id, or similar that groups pixels with similar properties). If input feature is None, than each pixel is randomly assigned to one of the target sets.

Target sets are defined by a list of ...


At the end I would add an example of definition of bins for train/test split in 80/20 and train/val/test split in 60/20/20.

Reply:
While writing above documentation it became clear to me that we don't always have polygon ids or similar feature that groups pixels. In that case it would be nice to be able to split pixels randomly. I understand that this is a new feature and now may not be the right time to implement it. I leave it up to you to decide.

Reply:
Updated docs and implemented random pixel assignment to train subsets.

Reply:
> While writing above documentation it became clear to me that we don't always have polygon ids or similar feature that groups pixels. In that case it would be nice to be able to split pixels randomly. I understand that this is a new feature and now may not be the right time to implement it. I leave it up to you to decide.

@azupanc By the way, this is exactly what I suggested in [this conversation](https://github.com/sentinel-hub/eo-learn/pull/117#discussion_r312134825). But probably my explanation wasn't clear enough.